style.css

```diff
@@ -707,6 +708,11 @@ h2 {
 
 /* ------------------------- Desktop Version ------------------- */
 @media screen and (min-width: 375px) {
```

Comment:
Dear Mehdi,

Please consider that we don't have one fixed size for mobile screens. Even if you check other phones in the dropdown list, you see different sizes. Just use iPhone 12 for example.

This task requires to make changes that only start affecting the design if screen size goes beyond 992px. You have used 375px which means the effect starts at that screen size. Please modify this line and use `min-width: 992px`. I know after this changes, there will be lots of other issues. Those issues should be addressed separately.

So in simple words, your styles should be the same unless the screen size is wider than 992px. This is what this task requires.